### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by smoyer64** [dennwc/dom#52]
There are now two forks on Github that have on-going maintenance while there's not really any activity here ... should I switch to one of the maintained projects?  As you've seen from my previous PRs, I'm not afraid to help with this project but you'd need to be around to at least merge (or reject) the PRs.

**Comment by redbar0n** [comment]
@dennwc A status update would be nice.

**Comment by smoyer64** [comment]
I've started using https://github.com/gowebapi/webapi instead which is generated from the Web IDL provided by Firefox.  I'm not a contributor to those projects and will be moving the test server from this project over there and making it a bit more configurable so that it's compatible with Skaffold and Ko.

**Comment by inliquid** [comment]
https://github.com/dominikh/go-js-dom
is in better state, at least it supports (v2) latest Go.

**Comment by dennwc** [comment]
Yeah, sorry, this project is slightly dead. I don't have time to maintain it at the moment :(

**Comment by dennwc** [comment]
If anyone is interested in taking it further, I can add a contributor or transfer an ownership.
